src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -607,13 +607,21 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     const deletes = objects.savedObjects.map((object) =>
       savedObjectsClient.delete(object.type, object.id, { force: true })
     );
-    await Promise.all(deletes);
-
-    // Unset this
-    this.setState({
-      selectedSavedObjects: [],
-    });
-
+    try {
+      await Promise.all(deletes);
+      // Unset this
+      this.setState({
+        selectedSavedObjects: [],
+      });
+    } catch (error) {
+      notifications.toasts.addDanger({
```

Comment:
Seems the new added code is not covered by unit test.

Reply:
I will need some unit tests.